src/installer/installer.go

```diff
@@ -276,6 +284,30 @@ func (i *installer) waitForBootkube(ctx context.Context) {
 	}
 }
 
+func (i *installer) waitForController() error {
+	i.log.Infof("Waiting for controller pod to start running")
+	err := i.ops.ReloadHostFile("/etc/resolv.conf")
+	if err != nil {
+		i.log.WithError(err).Error("Failed to reload resolv.conf")
+		return err
+	}
+	kc, err := i.kcBuilder(KubeconfigPath, i.log)
+	if err != nil {
+		i.log.WithError(err).Errorf("Failed to create kc client from %s", KubeconfigPath)
+		return err
+	}
+	controllerTimeout := time.Duration(i.Config.InstallationTimeout) * time.Minute
```

Comment:
timeout seems too long,
How long do you think we should wait?

Reply:
i think we will fail on timeout in assisted-service earlier. Didn't thought it is really big issue. If everything is ok we will have it running before bootkube finishes

Reply:
but if u think we need a smaller one, lets change just through the number

Reply:
If we expect it to run before bootkube finish, (and we call this function once bootkube is done) let's give it 5 minutes.

Reply:
sure, will change